{{cookiecutter.project_shortname}}/Dockerfile.dev.base

```diff
@@ -0,0 +1,8 @@
+{% include 'misc/header.py' %}
+FROM inveniosoftware/centos7-python:3.6
+
+COPY Pipfile Pipfile.lock ./
+COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
+
+RUN pipenv install --deploy --dev
+RUN pipenv run pip install --upgrade marshmallow==2.16.3
```

Comment:
This wouldn't be needed if there was an upper limit in one of the setup.py files
that include `marshmallow`, so it could be fixed like this?
```
~/invenio/my-site$ pipenv graph | grep marshmallow
  - marshmallow [required: >=2.7.0, installed: 3.0.0rc1]
    - marshmallow [required: >=2.15.2, installed: 3.0.0rc1]
  - marshmallow [required: >=2.5.0, installed: 3.0.0rc1]
    - marshmallow [required: >=2.15.2, installed: 3.0.0rc1]
```

The error that caused this is the following from `invenio-oaiserver`:
```
File "/root/.local/share/virtualenvs/src--FJPeFkS/lib/python3.6/site-packages/invenio_oaiserver/verbs.py", line 59, in DateTime
    _DateTime.DATEFORMAT_DESERIALIZATION_FUNCS,
AttributeError: type object 'DateTime' has no attribute 'DATEFORMAT_DESERIALIZATION_FUNCS'
```